src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -148,7 +149,7 @@ def post(self, request, project):
         if euser:
             report['event_user_id'] = euser.id
 
-        event = Event.objects.from_event_id(report['event_id'], project.id)
+        event = eventstore.get_event_by_id(project.id, report['event_id'])
```

Comment:
This may be a use case where passing some additional columns would be useful:
here we use 'environment', which is not in minimal column and that will trigger the nodestore fetch. 
I would consider:
1) double check that we would not load from nodestore if we only tried to fetch the three columns we need from Snuba
2) verify with james whether he prefers skipping the nodestore load and load one additional column from snuba

Reply:
I think this is loading from the `Environment` table rather than nodestore which we'll want to keep for now. I looked at this code again and realised we should to make a number of other changes to user reports which won't be easily tackled in this PR though.